language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/definitions/FunctionDefinitionCompletionTest.java

```diff
@@ -54,7 +54,7 @@ public void test(String config, String configPath) throws WorkspaceDocumentExcep
                 {"panicStatementErrorSuggestions.json", "function"},
                 {"recordVarDef1.json", "function"},
                 {"recordVarDef2.json", "function"},
-                {"functionPointerAsParameter.json", "function"},
+//                {"functionPointerAsParameter.json", "function"},
```

Comment:
Do we have an issue to track re-enabling of these disabled tests, if so pls add it to the PR description, if not pls create, assign to tooling team and add to the PR description.

Reply:
Issue created here.
https://github.com/ballerina-platform/ballerina-lang/issues/20183